include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -1650,15 +1650,15 @@ def AIE_ObjectFifoCreateOp: AIE_Op<"objectfifo", [HasParent<"DeviceOp">, Symbol]
 
     The following example shows an objectFifo which transposes a 16x16 matrix of
     `i32`s on the producer side using strides and sizes. No transformation is
-    applied on the consumer side of `%tile13` (in this case the `fromStream`
+    applied on the consumer side of `%tile13` (in this case the `dimensionsFromStreamPerConsumer`
     attribute may also be left off), and a transformation on `%tile23` first gives
     all even indices from the stream, followed by all odd indices:
 
     ```
-      aie.objectfifo @of4 (%tile12 toStream [<16, 1>, <16, 16>, <1,1>],
+      aie.objectfifo @of4 (%tile12 dimensionsToStream [<16, 1>, <16, 16>, <1,1>],
```

Comment:
The original keyword "toStream" was correct: you can see the custom parser for it [here](https://github.com/Xilinx/mlir-aie/blob/main/lib/Dialect/AIE/IR/AIEDialect.cpp#L517).

Reply:
For consistency, as discussed offline, I updated the parser